groundzero/src/constants/index.js

```diff
@@ -489,6 +489,38 @@ export const mentors = [
 
 ];
 
+<<<<<<< HEAD
```

Comment:
I think there are merge conflict markers here that may not have been removed, and two others in line 523 and 543. Perhaps these markers can be removed before merging?